.github/workflows/ci.yml

```diff
@@ -7,6 +7,9 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
```

Comment:
move this within the scope of the specific step, to limit these permissions to the minimum necessary to make this work

Reply:
Permissions still can't be set at a step level (only at job-level or globally). However, as we discussed, I've moved the permissions section to the top of its job. Thanks! 🙇🏻‍♂️